tests/profilefields/manager_test.php

```diff
@@ -0,0 +1,137 @@
+<?php
+/**
+*
+* This file is part of the phpBB Forum Software package.
+*
+* @copyright (c) phpBB Limited <https://www.phpbb.com>
+* @license GNU General Public License, version 2 (GPL-2.0)
+*
+* For full copyright and license information, please see
+* the docs/CREDITS.txt file.
+*
+*/
+
+class manager_test extends phpbb_database_test_case
+{
+	/** @var \phpbb\db\driver\driver_interface */
+	protected $db;
+
+	/** @var \phpbb\config\db_text */
+	protected $config_text;
+
+	/** @var \phpbb\profilefields\manager */
+	protected $manager;
+
+	/** @var \phpbb\log\log_interface */
+	protected $log;
+
+	/** @var \phpbb\db\tools */
+	protected $db_tools;
+
+	public function getDataSet()
+	{
+		return $this->createXMLDataSet(dirname(__FILE__).'/fixtures/manager.xml');
+	}
+
+	public function setUp()
+	{
+		parent::setUp();
+
+		global $phpbb_log;
+
+		$this->log = $this->prophesize('\phpbb\log\log_interface');
+		$phpbb_log = $this->log->reveal();
```

Comment:
@Nicofuma Trying to understand these tests but I have been unable to see an example of `prophesize` being used elsewhere.  Seems to me like this is brand new code...  If you point me to some documentation, I will try to find why this is failing to initialize (the travis errors come from the log setup area).